server/sonar-server/src/main/java/org/sonar/server/ce/ws/ActivityAction.java

```diff
@@ -160,19 +158,19 @@ public void handle(Request wsRequest, Response wsResponse) throws Exception {
   }
```

Comment:
not necessarily. Goal of this ticket is to prevent conflicts between organizations. The required permission, as specified in documentations, must be verified on the correct organization.